library/kotlin/src/io/envoyproxy/envoymobile/BUILD

```diff
@@ -40,17 +40,34 @@ envoy_mobile_kt_library(
         "EnvoyClientBuilder.kt",
         "EnvoyError.kt",
         "EnvoyStreamEmitter.kt",
+        "Filter.kt",
+        "FilterDataStatus.kt",
+        "FilterHeaderStatus.kt",
+        "FilterRegistry.kt",
+        "FilterTrailerStatus.kt",
         "GRPCClient.kt",
         "GRPCRequestBuilder.kt",
         "GRPCResponseHandler.kt",
         "GRPCStreamEmitter.kt",
         "HTTPClient.kt",
+        "Headers.kt",
+        "HeadersBuilder.kt",
         "LogLevel.kt",
         "Request.kt",
         "RequestBuilder.kt",
+        "RequestFilter.kt",
+        "RequestFilterCallbacks.kt",
+        "RequestHeaders.kt",
+        "RequestHeadersBuilder.kt",
         "RequestMapper.kt",
         "RequestMethod.kt",
+        "RequestTrailersBuilder.kt",
+        "RequestTrailers.kt",
+        "ResponseFilter.kt",
+        "ResponseFilterCallbacks.kt",
         "ResponseHandler.kt",
+        "ResponseHeaders.kt",
```

Comment:
We are missing builders for ResponseHeaders/Trailers for both kotlin and swift.

Reply:
Ah yea we should add those. Originally I thought they wouldn't be necessary, but since we'll allow filters to add/mutate response headers/trailers, we'll need to add those. Can do in a separate PR (maybe comment on the issue?)